MetaTempDriver/Hardware/library/KelvinVR/KelvinVR.h

```diff
@@ -0,0 +1,41 @@
+/*
+  Main class for KelvinVR module
+*/
+
+#if defined(__AVR__)
+	#include "Arduino.h"
+	#include "hardware/avr/HW_AVR_defines.h"
+#elif defined(__PIC32MX__)
+	#include "WProgram.h"
+	#include "hardware/pic32/HW_PIC32_defines.h"
+#elif defined(__arm__)
```

Comment:
[nit] Create support for ESP32 Platform in the future

Reply:
Created new files for ESP32 and Arduino